.travis.yml

```diff
@@ -161,6 +161,17 @@ jobs:
       env: TAV=knex,ws,graphql,express-graphql
       script: tav --quiet
 
+    # Node.js 0.10
+    - stage: 'test'
```

Comment:
Have this been placed last on purpose? Since the `test` stage always will come before any other stage, it would make it easier to read if this was moved up above the `dependencies` stage